src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/adapters/AbstractTableAdapter.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import {computed} from 'mobx';
+import React from 'react';
+import Table from '../../../components/Table';
+import datagridFieldTransformerRegistry from '../registries/DatagridFieldTransformerRegistry';
+import type {Schema} from '../types';
+import AbstractAdapter from './AbstractAdapter';
+
+export default class AbstractTableAdapter extends AbstractAdapter {
+    static defaultProps = {
+        data: [],
+    };
+
+    @computed get data(): Array<Object> {
+        return this.props.data;
```

Comment:
Why can't we simply use the `this.props.data` observable instead of creating a separate computed value?